tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Install mumble-server
+  apt:
+    name: mumble-server
+  notify:
+    - Enable and restart mumble-server
+
+- name: Instantiate mumble-server config
+  vars:
+    mumble_server_config: "{{ default_mumble_server_config | combine(mumble_server.config | default({}))}}"
+    mumble_server_ice_config: "{{ default_mumble_server_ice_config | combine(mumble_server.ice_config | default({}))}}"
+  template:
+    src: mumble-server.ini.j2
+    dest: /etc/mumble-server.ini
+  notify:
+    - Enable and restart mumble-server
+
+- name: Set superuser password
+  command:
```

Comment:
Ummmmmmmmmmmmmm

```
command: murmurd -supw {{ … }}
become_user: murmurd
```

Reply:
`no_log: True`

Reply:
ehm right...

Reply:
> `no_log: True`

This breaks the failed_when command

Reply:
> > `no_log: True`
> 
> This breaks the failed_when command

Never mind, tried no_log under command instead of under name ...